addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -680,6 +680,13 @@ registry.backgroundVideo = publicWidget.Widget.extend({
 
         $(window).on('resize.' + this.iframeID, throttledUpdate);
 
+        let sPopup = this.$target.closest('.s_popup');
+        if (sPopup.length) {
+            sPopup.on('shown.bs.modal', () => {
+                this._adjustIframe();
+                this.$target.find('.o_bg_video_container').show();
```

Comment:
There are a couple of reasons for writing here. The foremost one is that it will flicker while the popup open. Another one is in s_popup js I have to check `bgVideoContainer` length so code will be twice there.